src/css/controls/imports/playback-label.less

```diff
@@ -1,8 +1,12 @@
 @import "../../shared-imports/vars.less";
 
+.jw-idle-icon-text {
+    display: none;
```

Comment:
Can we move the styling (except `display: block`) from lines 21–26 here? This will make it easier to re-use the class when the player is not idle (`.jw-state-idle`). We can even remove the `display: block`, and control this on `.jw-idle-label` instead.